iocage/cli/create.py

```diff
@@ -42,10 +48,8 @@ def validate_count(ctx, param, value):
 
             return int(value)
         except ValueError:
-            ioc_common.logit({
-                "level"  : "EXCEPTION",
-                "message": f"{value} is not a valid integer."
-            })
+            logger.log.error(f"{value} is not a valid integer.")
```

Comment:
While you are at it please check other places too. A quick hack that has usually helped me is `git grep logger.log` from the top level dir of your git repo and it will list you repowide occurences of this. Hope this helps.